### ISSUE TITLE
Able to move bolted down AI as borg.

### ISSUE THREAD

**Issue by GrimsonGrime** [tgstation/tgstation#70372]

**Comment by msgerbs** [comment]
It's not just borgs. I was an AI last night and had my bolts down and... something? moved me to the side a tile. I believe grav was off so maybe it's an issue when gravity is off.

**Comment by zkxs** [comment]
I don't think this is gravity dependent. I believe that anyone on help intent can switch positions with the AI simply by walking into it, regardless of the AI's floor bolt state.

I've seen this happen multiple times the past few days, and it's been caused by both humans and borgs moving into the AI.

**Comment by msgerbs** [comment]
I wasn't even switching places with anyone. I believe an explosion pushed me. I was being attacked by miners with mining bombs and gravity was down. I suddenly noticed I had moved out of my usual AI perch, and double checked my bolts were down (which they were).
